go.mod

```diff
@@ -6,16 +6,17 @@ require (
 	github.com/ghodss/yaml v1.0.0 // indirect
 	github.com/gogo/protobuf v1.2.1 // indirect
 	github.com/golang/glog v0.0.0-20160126235308-23def4e6c14b // indirect
-	github.com/golang/protobuf v1.3.2 // indirect
+	github.com/golang/protobuf v1.3.1 // indirect
 	github.com/google/btree v1.0.0 // indirect
 	github.com/google/go-cmp v0.3.0 // indirect
-	github.com/google/go-containerregistry v0.0.0-20190723205234-74f764b9c575 // indirect
+	github.com/google/go-containerregistry v0.0.0-20190623150931-ca8b66cb1b79 // indirect
+	github.com/google/gofuzz v1.0.0 // indirect
 	github.com/googleapis/gnostic v0.3.0 // indirect
 	github.com/gregjones/httpcache v0.0.0-20190611155906-901d90724c79 // indirect
 	github.com/hashicorp/golang-lru v0.5.1 // indirect
 	github.com/imdario/mergo v0.3.7 // indirect
 	github.com/inconshreveable/mousetrap v1.0.0 // indirect
-	github.com/json-iterator/go v1.1.7 // indirect
+	github.com/json-iterator/go v1.1.6 // indirect
```

Comment:
downgrade ?

Reply:
I have no idea, I just let `./hack/build.sh` do its thing.

Reply:
Maybe a rebase is needed ?

Reply:
Should be sorted now.